### ISSUE TITLE
Landing page https://flathub.org shows an NGINX development page

### ISSUE THREAD

**Issue by nbenitez** [flathub/linux-store-frontend#273]
As of a few days, the main landing page https://flathub.org just shows an NGINX development page, it's not showing the corresponding web application landing page.

Issue can be seen by just visiting https://flathub.org

**Comment by AsciiWolf** [comment]
Hmm, I am not able to reproduce this issue. Correct Flathub page is loaded for me.

**Comment by nbenitez** [comment]
Well I just tried, see image below:

![Captura de pantalla de 2021-01-17 13-35-46](https://user-images.githubusercontent.com/580854/104851086-2d808400-58c9-11eb-8c56-97bca73f2d9f.png)

**Comment by nbenitez** [comment]
I'm connecting from Santo Domingo, Dominican Republic. I just look up my IP and it is `148.103.196.138`, it could be a routing or firewall rules problem.

**Comment by nbenitez** [comment]
The problem happens only with the mentioned url, because for example https://flathub.org/home loads fine and I can browse the rest of the site from there, but when browsing https://flathub.org/ it shows the error mentioned.

**Comment by nbenitez** [comment]
Ok now solved!!, problem was on my side. It seems I got to visit https://flathub.org/ at some point in the past where it showed that nginx page (maybe in the middle of a site update) and Firefox cached that, and that's why it was showing me that all the time. I now forced a page refresh and it redirected me to https://flathub.org/home which I think it's the expected behaviour.

Sorry for the trouble, I proceed to close the issue.
